.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+concurrency:
+  group: ${{ github.ref_name }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  rubocop:
+    name: Rubocop
+    uses: theforeman/actions/.github/workflows/rubocop.yml@v0
+    with:
+      command: bundle exec rubocop --parallel --format github
+
+  test:
+    name: Ruby Tests
+    needs: rubocop
+    uses: theforeman/actions/.github/workflows/foreman_plugin.yml@v0
+    with:
+      plugin: foreman_plugin_template
+
+  rename_test:
+    name: Rename Script Test
+    needs: [rubocop, test]
+    runs-on: ubuntu-latest
+
+    env:
+      PLUGIN_NAME: foreman_plugin_template
```

Comment:
This is exactly the same name as the template, so a rename is not going to do anything.